index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/home/safeez/Desktop/Portfolio/Portfolio-Website/style.css">
```

Comment:
- [ ] [OPTIONAL] You can reference the stylesheet like below. Since it is the same level as this file
```
<link rel="stylesheet" href="./style.css">
```